docs/specs/bundle-specs.md

```diff
@@ -28,3 +28,14 @@ apps:
 The list of apps to be run by `dusty up` if this bundle is activated.
 These apps are used as entrypoints into the dependency graph defined by
 your app and service specs.
+
+## services
+
+```
+services:
+  - postgres
+```
+
+An optional list of services which will be run by `dusty up` if this bundle
```

Comment:
The use of the word "optional" here makes it seem like apps are mandatory. The actual constraint is that at least one app or service must be provided, right?